docs/rules/no-invalid-model-keys.md

```diff
@@ -0,0 +1,108 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-invalid-model-keys
+description: require valid keys in model prop
+since: v3.9.0
+---
+# vue/no-invalid-model-keys
+
+> require valid keys in model prop
+
+- :gear: This rule is included in all of `"plugin:vue/vue3-essential"`, `"plugin:vue/essential"`, `"plugin:vue/vue3-strongly-recommended"`, `"plugin:vue/strongly-recommended"`, `"plugin:vue/vue3-recommended"` and `"plugin:vue/recommended"`.
```

Comment:
Do not change the shareable configuration until a major version upgrade. Would you please remove this wording?

```suggestion
```